services/QuillCMS/Gemfile

```diff
@@ -8,8 +8,8 @@ git_source(:github) do |repo_name|
 end
 
 # CORE
-gem 'puma', '~> 4.3'
-gem 'rails', '~> 6.1'
+gem 'puma', '~> 4.3.11'
+gem 'rails', '~> 6.1.4.6'
```

Comment:
I don't think it's necessary to make any changes to the Gemfile here.  Instead, you can run `bundle update rails scenic puma` from the CMS directory and get all of the same updates since the dependencies should get pulled in automatically.  This keeps the version listing numbers at minor release instead of patch versions.